docs/reference/objects/series.md

```diff
@@ -47,3 +47,7 @@ Returns an array of published [products]({% link docs/reference/objects/product/
 
 Returns an array of published [products]({% link docs/reference/objects/product/index.md %}) in the series which are taking place currently or are upcoming.
 
+# series.featured_variant
+
+Returns the cheapest [variant]({% link docs/reference/objects/product/variant/index.md %}) in the series, factoring in display variants and promotions.
```

Comment:
The featured variant is not necessarily the cheapest, and I'm not sure this is description makes that clear?
This method end up using the following method with a maybe more descriptive comment.
https://github.com/easolhq/easol/blob/4cee1c93528223e36de9c465741534a41bbb8f20/app/models/concerns/listing/priceable.rb#L28 that have a more accurate message.

Reply:
Ah good point! I misread the doc on the drop itself 👍